main.py

```diff
@@ -45,7 +46,9 @@ def create_totalmobile_jobs_trigger(_event, _context) -> str:
 
     questionnaire_service = QuestionnaireService(
         blaise_service=RealBlaiseService(config),
-        eligible_case_service=EligibleCaseService(),
+        eligible_case_service=EligibleCaseService(
```

Comment:
Since this is construct in both main.py and endpoints.py, maybe a good time to start creating some shared factory functions somewhere?

Reply:
Yes we plan to going forward